bot/exts/utils/bot.py

```diff
@@ -44,6 +44,8 @@ async def echo_command(self, ctx: Context, channel: Optional[TextChannel], *, te
         """Repeat the given message in either a specified channel or the current channel."""
         if channel is None:
             await ctx.send(text)
+        elif not channel.permissions_for(ctx.author).send_messages:
+            await ctx.send('You don\'t have permission to speak in that channel.')
```

Comment:
```suggestion
            await ctx.send("You don't have permission to speak in that channel.")
```
What sort of escapism is this? 🦖 🦖🦖